common/formatting/token_partition_tree.cc

```diff
@@ -535,65 +540,126 @@ static bool AppendFittingSubpartitions(
   int indent;
 
   // Try appending first argument
-  const auto& first_arg = args[0];
-  auto group_value_arg = group->Value().Value(first_arg);
-  if (wrap_first_subpartition ||
-      (FitsOnLine(group_value_arg, style).fits == false)) {
-    // Use wrap indentation
-    indent = style.wrap_spaces + group_value_arg.IndentationSpaces();
 
-    // wrap line
-    group = group->NewSibling(first_arg.Value());  // start new group
-    child = group->NewChild(first_arg);  // append not fitting 1st argument
-    group->Value().SetIndentationSpaces(indent);
+  const TokenPartitionTree& first_arg = subpartitions.front();
+  verible::UnwrappedLine first_line = group->Value().Value(first_arg);
+  if (trailer && subpartitions.size() == 1) {
+    first_line.SpanUpToToken(trailer->Value().TokensRange().end());
+  }
 
-    // Wrapped first argument
-    wrapped_first_subpartition = true;
-  } else {
+  bool wrapped_first_subpartition;
```

Comment:
Below, this value is essentially depending on the result of the `!wrap_first_subpartition && (fit_result = FitsOnLine(first_line, style)).fits == true)` if condition (if I see it correctly).
So maybe we can assign the value direclty and make it const.
```
  const bool wrapped_first_subparition = ....
```

Reply:
Done